contributors/guide/pull-requests.md

```diff
@@ -590,6 +590,26 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
+## Fixing linter issues
+
+Kubernetes has a set of linter checks. Some of those must pass in the entire
+code base, some must pass in new or modified code, and some are merely hints
+to developers how to improve their code.
+
+Please do not create Pull Requests for issues found by linters without first
+reaching out to maintainers to determine whether there is sufficient interest
```

Comment:
> reaching out to maintainers

do we have a recommended way to do that?

Reply:
Not sure. Shall we suggest to bring it up on the #code-organization Slack channel?

Reply:
I guess, probably?

Reply:
That seems as close as we have to offer

Reply:
Added.